src/queryParameters 2.js

```diff
@@ -0,0 +1,27 @@
+import { useHistory, useLocation } from "react-router-dom";
```

Comment:
This file is already on the master branch. I think it should be removed before the merge.